newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/context/InstrumentationContextManager.java

```diff
@@ -221,11 +220,6 @@ public boolean shouldTransform(final String internalClassName, final ClassLoader
             Agent.LOG.log(Level.FINEST, "Skipping transform of {0}. Classloader {1} is excluded.", internalClassName, classloader);
             return false;
         }
-        if (internalClassName.startsWith("javax/crypto/")) {
-            // crypto classes can cause class circularity errors if they get too far along in the class transformer
-            Agent.LOG.finest(MessageFormat.format("Instrumentation skipped by ''javax crypto'' rule: {0}", internalClassName));
-            return false;
-        }
```

Comment:
This exclude rule was moved to the `META-INF/excludes` file to simplify the logic and keep all of the security agent related excludes in a central place.